tests/testthat/testImportCNA.GISTIC2.R

```diff
@@ -0,0 +1,45 @@
+#
+# test importCNA.GISTIC2.R
+#
+library(testthat)
```

Comment:
never explicitly load libraries in package code. This is handled by the package loader.